shared/src/main/kotlin/com/egm/stellio/shared/util/AuthUtils.kt

```diff
@@ -65,24 +59,9 @@ object AuthContextModel {
     const val AUTH_PROP_FAMILY_NAME = AUTHORIZATION_ONTOLOGY + AUTH_TERM_FAMILY_NAME
     const val AUTH_TERM_SAP = "specificAccessPolicy"
     const val AUTH_PROP_SAP = AUTHORIZATION_ONTOLOGY + AUTH_TERM_SAP
-    const val AUTH_TERM_RIGHT = "right"
-    const val AUTH_PROP_RIGHT: ExpandedTerm = AUTHORIZATION_ONTOLOGY + AUTH_TERM_RIGHT
-    const val AUTH_TERM_IS_DELETED = "isDeleted"
-    const val AUTH_PROP_IS_DELETED: ExpandedTerm = AUTHORIZATION_ONTOLOGY + AUTH_TERM_IS_DELETED
 
     const val AUTH_TERM_IS_MEMBER_OF = "isMemberOf"
     const val AUTH_REL_IS_MEMBER_OF: ExpandedTerm = AUTHORIZATION_ONTOLOGY + AUTH_TERM_IS_MEMBER_OF
-    const val AUTH_TERM_CAN_READ = "canRead"
-    const val AUTH_REL_CAN_READ: ExpandedTerm = AUTHORIZATION_ONTOLOGY + AUTH_TERM_CAN_READ
-    const val AUTH_TERM_CAN_WRITE = "canWrite"
-    const val AUTH_REL_CAN_WRITE: ExpandedTerm = AUTHORIZATION_ONTOLOGY + AUTH_TERM_CAN_WRITE
-    const val AUTH_TERM_CAN_ADMIN = "canAdmin"
-    const val AUTH_REL_CAN_ADMIN: ExpandedTerm = AUTHORIZATION_ONTOLOGY + AUTH_TERM_CAN_ADMIN
-    const val AUTH_TERM_IS_OWNER = "isOwner"
-    const val AUTH_REL_IS_OWNER: ExpandedTerm = AUTHORIZATION_ONTOLOGY + AUTH_TERM_IS_OWNER
-    val ALL_IAM_RIGHTS_TERMS = setOf(AUTH_TERM_CAN_READ, AUTH_TERM_CAN_WRITE, AUTH_TERM_CAN_ADMIN, AUTH_TERM_IS_OWNER)
-    val ALL_IAM_RIGHTS = setOf(AUTH_REL_CAN_READ, AUTH_REL_CAN_WRITE, AUTH_REL_CAN_ADMIN, AUTH_REL_IS_OWNER)
-    val ALL_ASSIGNABLE_IAM_RIGHTS = ALL_IAM_RIGHTS.minus(AUTH_REL_IS_OWNER)
 
     enum class SpecificAccessPolicy {
```

Comment:
I guess it should disappear in this PR

Reply:
or partly (we still need a way to set something like this at entity creation time)